skills/morph-ppt/reference/styles/vivid--candy-stripe/style.md

```
@@ -0,0 +1,73 @@
+| `!!stripe-red`    | rect | Red full-width rainbow stripe    |
+| `!!stripe-orange` | rect | Orange full-width rainbow stripe |
+| `!!stripe-yellow` | rect | Yellow full-width rainbow stripe |
+| `!!stripe-green`  | rect | Green full-width rainbow stripe  |
+| `!!stripe-blue`   | rect | Blue full-width rainbow stripe   |
+| `!!stripe-purple` | rect | Purple full-width rainbow stripe |
+
+## Page Structure (5 pages)
+
+| Slide | Type      | Elements                                                                                                 | Description |
+| ----- | --------- | -------------------------------------------------------------------------------------------------------- | ----------- |
+| S1    | hero      | Cover — 6 rainbow stripes evenly distributed (3.4cm spacing), centered title                             |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)